docs/hub/repositories-licenses.md

```diff
@@ -62,6 +62,7 @@ A full list of the available licenses is available here:
 | ISC                                                            | `isc`                                    |
 | LaTeX Project Public License v1.3c                             | `lppl-1.3c`                              |
 | Microsoft Public License                                       | `ms-pl`                                  |
+| Apple ML-MobileCLIP License                                    | `ml-mobileclip`                          |
```

Comment:
This is not a mobileCLIP specific license, it's a more general one. Apple folks called it `apple-ascl` at https://huggingface.co/apple/MobileCLIP-S1

Reply:
Indeed, it's the "[Apple Sample Code License](https://developer.apple.com/support/downloads/terms/apple-sample-code/Apple-Sample-Code-License.pdf)".

Reply:
I could swear I saw ML-MobileCLIP license (which is MIT license with a disclaimer for distribution at the bottom) in TiC-CLIP repos. I'll fix thank you for the heads-up

Reply:
Fixed and added file